parts/linux/cloud-init/artifacts/components.json

```diff
@@ -70,7 +70,8 @@
           "latestVersion": "1.8.22",
           "previousLatestVersion": "1.8.20"
         }
-      ]
+      ],
+      "windowsVersions": []
```

Comment:
do we need to always explicitly declare an empty set of windows versions?

Reply:
Yes, because otherwise we fallback to the "multiArch" versions and the windows VHDs blow up.